modules/core/src/lib/loader-utils/loggers.ts

```diff
@@ -1,4 +1,7 @@
 // probe.gl Log compatible loggers
+import {Log} from 'probe.gl';
+
+export const probeLog = new Log({id: 'loaders.gl'});
 
 // Logs nothing
 export class NullLog {
```

Comment:
Do we still need the `ConsoleLog` class in this file?

Reply:
I don't want to change that much right now. I will do another pass later.